README.md

```diff
@@ -296,9 +296,26 @@ You can use `config.getRemote()` and `config.setRemote()` for
 
 `npm test`
 
+## Best Practices
+
+Zero-config is designed to help you structure your config 
+files to support a number of production concerns. These best
+ practices reflect our approach and some of the reasons we 
+ designed Zero-config as we did.
+
+ - Most configuration should live in `common.json`
```

Comment:
let's just rephrase it `should live in a single file`

Let's do the breaking change of production vs common some other time as its a nightmare change ;_;